bundles/org.openhab.binding.onewire/src/main/feature/feature.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<features name="org.openhab.binding.onewire-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
-    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
+<features name="org.openhab.binding.onewire-${project.version}"
+	xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
```

Comment:
Best put tags on a single line in feature files.

Reply:
I think this is the result of the formatted. But I‘ll change it.